packages/arb-bridge-eth/contracts/bridge/Outbox.sol

```diff
@@ -134,9 +134,9 @@ contract Outbox is IOutbox, Cloneable {
      * @notice Executes a messages in an Outbox entry.
      * @dev Reverts if dispute period hasn't expired, since the outbox entry
      * is only created once the rollup confirms the respective assertion.
-     * @param batchNum Index of OutboxEntry in outboxEntries array
+     * @param batchNum Number of target OutboxEntry
```

Comment:
Isn't this actually the `Index of OutboxEntry in outboxEntries array` as the previous doc?
I don't think `Number of target OutboxEntry` is clearer.

Maybe we can call it the identifier of the batch?

Reply:
yeah not sure why i did this tbh